rmw_fastrtps_cpp/include/rmw_fastrtps_cpp/MessageTypeSupport.h

```diff
@@ -3,19 +3,41 @@
 
 #include "TypeSupport.h"
 #include "rosidl_typesupport_introspection_cpp/message_introspection.hpp"
+#include "rosidl_typesupport_introspection_cpp/field_types.hpp"
+
+#include <fastcdr/FastBuffer.h>
+#include <fastcdr/Cdr.h>
```

Comment:
The package needs to export the FastCDR include directories in order to make them available to downstream packages.